backend/rates/rates.go

```diff
@@ -73,10 +73,10 @@ func (updater *RateUpdater) update() {
 		strings.Join(coins, ","),
 		strings.Join(fiats, ","),
 	)
-	updater.log.Infof("Rates url: %v", url)
 	response, err := client.Get(url)
 	if err != nil {
-		updater.log.Printf("Error getting rates: %v\n", err)
+		updater.log.Errorf("Error getting rates: %v\n", err)
+		updater.log.Infof("Rates url: %v", url)
```

Comment:
I guess it's ok to log it separately but the idea is to log the URL only when an error happens. So, they kind of logically tied together. I would use a single log statement:

```go
updater.log.Errorf("Error getting rates from %q: %v", url, err)
```